### DISCUSSION TITLE
`INSERT` multiple JSON `CONTENT`s at once

### DISCUSSION THREAD

**Discussion by gramian** [ArcadeData/arcadedb#1226] [Ideas]
Dear All,
I would like to propose to extend the syntax of inserting JSON, ie:
```
INSERT INTO doc CONTENT { "name": "Jerry" };
```
such that it allows multiple record insertions in a single command, just like  `VALUES` does.
This could look like:
```
INSERT INTO doc CONTENT { "name": "Jerry" }, { "num": 4 }, { "id": "x1" };
```
which would result in three records.

**Comment by tolgaulas** [comment]
I think it should honor the json array structure like `... CONTENT [{ "name": "Jerry" }, { "num": 4 }, { "id": "x1" }]`

**Comment by lvca** [comment]
Ok, I like the json array format. Starting the implementation and tests.

**Comment by lvca** [comment]
Followup on #1257

**Comment by lvca** [comment]
Implemented in 23.10.1-SNAPSHOT.
